app/src/controllers/user/reducer.js

```diff
@@ -41,24 +41,26 @@ export const settingsReducer = (state = SETTINGS_INITIAL_STATE, { type, payload
 export const userAssignedProjectReducer = (state = {}, { type, payload }) => {
   switch (type) {
     case ASSIGN_TO_RROJECT_SUCCESS: {
-      const { projectName, projectRole, entryType } = payload;
+      const { projectKey, projectRole, entryType, organizationSlug, projectName } = payload;
```

Comment:
Do you provide `organizationSlug` when dispatching `ASSIGN_TO_RROJECT_SUCCESS` action?
I cannot find it.

Reply:
![screenshot_08](https://user-images.githubusercontent.com/84175555/196341621-d9006677-967d-4002-be0d-5e6a75df6993.jpg)
I provide it from addProjectSaga.
But I see, I forgot provide it from another place.